lib/committee/rails/test/methods.rb

```diff
@@ -5,12 +5,20 @@ module Methods
 
       def committee_schema
         @committee_schema ||= begin
+          schema = committee_options[:schema]
+
+          return schema if schema
```

Comment:
I prefer to move these additional lines between L6 and L7 or assigns `committee_options[:schema]` to `@committee_schema` for code consistency.